src/cli/commands/retrieve_metrics.go

```diff
@@ -15,7 +15,7 @@ type MetricsCommand struct {
 	RequiredlArgs MetricsPositionalArgs `positional-args:"yes"`
 	StartTime     string                `long:"start" description:"start time of metrics collected with format \"yyyy-MM-ddTHH:mm:ss+/-HH:mm\" or \"yyyy-MM-ddTHH:mm:ssZ\", default to very beginning if not specified."`
 	EndTime       string                `long:"end" description:"end time of the metrics collected with format \"yyyy-MM-ddTHH:mm:ss+/-HH:mm\" or \"yyyy-MM-ddTHH:mm:ssZ\", default to current time if not speficied."`
-	Desc          bool                  `long:"desc" description:"display in descending order, default to ascending order if not specified."`
+	Asc           bool                  `long:"asc" description:"display in ascending order, default to descending order if not specified."`
```

Comment:
Some users are already using "--desc" flag in their scripts.  Then,  with this change,  the original script will fail with 
```
>> cf asm APPNAME memoryused --desc
FAILED

Error: unknown flag `desc'
```
How about keeping the "Desc" flag definition here to make the above cmd are still work ?  
For Metadata and documents,  we can replace "desc"    to "asc". 

Anyway, the cmd "cf asm APPNAME memoryused --desc --asc"  will be acceptable as well according to above suggestion , but only "--asc" take effect here.

Reply:
that's better

Reply:
@boyang9527 fixed in the commit